src/main/cli/settings.c

```diff
@@ -1261,6 +1261,12 @@ const clivalue_t valueTable[] = {
     { PARAM_NAME_EZ_LANDING_THRESHOLD,      VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, ez_landing_threshold) },
     { PARAM_NAME_EZ_LANDING_LIMIT,          VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 75 }, PG_PID_PROFILE, offsetof(pidProfile_t, ez_landing_limit) },
 
+#ifdef USE_THRUST_IMBALANCE_DETECTION
+    { PARAM_NAME_THRUST_IMBALANCE_THRESHOLD,       VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_PID_PROFILE, offsetof(pidProfile_t, thrust_imbalance_threshold) },
+    { PARAM_NAME_THRUST_IMBALANCE_TRIGGER_DELAY,   VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_PID_PROFILE, offsetof(pidProfile_t, thrust_imbalance_trigger_delay) },
+    { PARAM_NAME_THRUST_IMBALANCE_UNTRIGGER_DELAY, VAR_UINT8 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_PID_PROFILE, offsetof(pidProfile_t, thrust_imbalance_untrigger_delay) },
```

Comment:
IS there any reason to fine-tune this value?

Reply:
Actually I think there isn't a good one. What value you'd say would be optimal to be the hardcoded default? 1 sec?

Reply:
1 sec sounds good. And the logic may be updated so that imbalance is reset always after this time, triggering again shortly after if it persists. Frequency of retrigger may indicate how bad is the situation (current code is  very 'latching' when noise is present - multiple consecutive samples must be positive for trigger, them multiple negative ones are necessary for untrigger)